index.js

```diff
@@ -1,21 +1,42 @@
+/*jshint node: true */
+'use strict';
+
 var path = require('path');
+var mergeTrees = require('broccoli-merge-trees');
+var Funnel = require('broccoli-funnel');
 
 module.exports = {
   name: 'ember-data-factory-guy',
 
+  treeForVendor: function() {
```

Comment:
Yes they are included in the build .. they are put into tests.js 
 and that file is added when tests/index.html includes it as a script tag.
And it would be simple to just include that too in the app/index.html file BUT
 that tests.js file also has the test-helper.js file which requires ember-qunit ( and that is not available to development build .. not sure why?? but the easy answer would be to include the tests.js in the contentFor 'bodyFooter'  and then figure out a way to make ember-qunit available to development .. just not sure how to do that
 or even better would be to make a file like tests.js but that does not have the tests and the test-helper and include that in the contentFor 'bodyFooter'